### DISCUSSION TITLE
Homogeneous Isotropic turbulence in a periodic box

### DISCUSSION THREAD

**Discussion by aeroamit** [su2code/SU2#2164] [Q&A]
How to carry out simulation for homogeneous Isotropic turbulence in a periodic box (using DES or LES) in SU2. 
Any ideas/setup?

Regards
Amit

**Comment by bigfooted** [comment]
I am not sure if this works... out of the box.
There is LES, and there are periodic BC's, but I do not think that periodic BC's are implemented in the discontinuous Galerkin method, which is the basis for the LES implementation. Maybe it is easy to re-use the periodic BC's from the Euler solver.

**Reply by aeroamit** [reply]
True... I thought if someone has already done, it will be an easy go. We don't see Eduardo active on GitHub nowadays? He may probably know (might have done already).

**Reply by bigfooted** [reply]
You could try asking @EduardoMolina . There might be some stuff hidden in a branch somewhere that he or others have worked on. You could also ask this question on slack, there are some other folks that are working on LES that are less active on github.

**Reply by EduardoMolina** [reply]
Hi Amit!
Sorry for the late response.
I never tried a homogeneous Isotropic turbulence using LES or DES but it should be relatively straightforward if you already have the correct initial conditions.
Best,
Eduardo

**Reply by aeroamit** [reply]
Hi @EduardoMolina , No probs. As you and Nisjo mentioned, it should be straight to setup with periodic BCs. But seems like periodic BCs have some issues (being worked out in some PR, if I am not wrong). Also, how can we initialize the volume region with specific conditions (some function). Probably we have to hard code?

Regards
Amit
